config.default.json

```diff
@@ -12,7 +12,7 @@
     "MAX_AGE_MS": 60000
   },
   "WORKLOADS_TO_SCAN_QUEUE_WORKER_COUNT": 10,
-  "METADATA_TO_SEND_QUEUE_WORKER_COUNT": 10,
+  "REQUEST_QUEUE_LENGTH": 2,
```

Comment:
Why it's 2?

Reply:
Assuming a response time of 200-300ms then 2 concurrent requests should average out to about 400-500 requests per minute. I think that is a decent starting point. We can tweak as we go